driver/src/price_finding/linear_optimization_price_finder.rs

```diff
@@ -72,39 +88,55 @@ fn serialize_balances(state: &models::AccountState, orders: &[models::Order]) ->
 fn serialize_order(order: &models::Order, id: &str) -> serde_json::Value {
     json!({
         "accountID": account_id(order.account_id),
-        "sellToken": token_id(order.sell_token as usize),
-        "buyToken": token_id(order.buy_token as usize),
+        "sellToken": token_id(order.sell_token),
+        "buyToken": token_id(order.buy_token),
         "sellAmount": order.sell_amount.to_string(),
         "buyAmount": order.buy_amount.to_string(),
         "ID": id //TODO this should not be needed
     })
 }
 
+fn parse_token(key: &str) -> Result<u16, PriceFindingError> {
+    if key.len() < 6 {
```

Comment:
This seems a bit hacky. Can we use `starts_with` or some other more explicit way to signal we expect the name to start with "token"

Reply:
Yes! Great! I was looking for something like this.